shared/source/command_container/cmdcontainer.cpp

```diff
@@ -525,6 +525,7 @@ void CommandContainer::fillReusableAllocationLists() {
         return;
     }
 
+    this->getResidencyContainer().reserve(amountToFill);
```

Comment:
auto reserveSize = this->useSecondaryCommandStream ? amountToFill * 2 : amountToFill;